app/bundles/CoreBundle/Assets/js/4.builder.js

```diff
@@ -1148,6 +944,14 @@ Mautic.initSlotListeners = function() {
                 slot.remove();
                 focus.remove();
             });
+            cloneLink.click(function(e) {   
+                var cloneBtn = mQuery(this);
+                var clonedElem = cloneBtn.closest('[data-slot]');
+                clonedElem.clone().insertAfter(clonedElem);
+                Mautic.initSlotListeners();
+                Mautic.initSections();
+                Mautic.initSlots();
```

Comment:
It technically works, but it's freezing my browser. It's doing some tasks that aren't necessary. For example we don't have to `initSections()` again since cloning a slot is not modifying sections. Lets remove that. The `initSlot` can send what slots to init in the attribute. Let's try that.